proxy/envoy/testdata/cb-policy.yaml.golden

```diff
@@ -2,10 +2,10 @@ destination: world.default.svc.cluster.local
 circuit_breaker:
   simple_cb:
     max_connections: 100
-    sleep_window_seconds: 15.5
+    sleep_window: 15.5s
```

Comment:
Any reason why this is specified in seconds vs ms? is dest-policy.yaml we use 15500ms for the duration.